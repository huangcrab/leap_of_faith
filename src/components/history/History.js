import React, { Component } from "react";
import Navbar from "../layout/Navbar";
import M from "materialize-css";
import "./history.css";
export default class History extends Component {
  constructor(props) {
    super(props);
  }
  componentDidMount() {
    M.AutoInit();
  }
  render() {
    return (
      <React.Fragment>
        <div className="row history">
          <Navbar active="history" logo={true} />
        </div>
        <div className="row history">
          <div className="col s12 m3 history-sections ">
            <div className="history-search history-container">
              <div className="env-pick">
                <div className="input-field col s12 env-select">
                  <select>
                    <option defaultValue="" disabled selected>
                      Choose your Environment
                    </option>
                    <option value="1">Option 1</option>
                    <option value="2">Option 2</option>
                    <option value="3">Option 3</option>
                  </select>
                </div>
              </div>
              {/* <div className="search-bar">search-bar</div>
              <div className="search-result">search-result</div> */}
            </div>
          </div>
          <div className=" col s12 m6 history-sections ">
            <div className="history-main history-container ">Search</div>
          </div>
          <div className=" col s12 m3 history-sections ">
            <div className="history-analytic history-container ">Search</div>
          </div>
        </div>
      </React.Fragment>
    );
  }
}
