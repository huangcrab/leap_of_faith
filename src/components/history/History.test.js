import React from "react";
import { shallow } from "enzyme";
import History from "./History";
describe("History", () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<History debug />);

    expect(component).toMatchSnapshot();
  });
});
