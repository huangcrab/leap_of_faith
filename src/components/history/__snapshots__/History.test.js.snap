// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History should render correctly in "debug" mode 1`] = `
<Fragment>
  <div
    className="row history"
  >
    <Navbar
      active="history"
      logo={true}
    />
  </div>
  <div
    className="row history"
  >
    <div
      className="col s12 m3 history-sections "
    >
      <div
        className="history-search history-container"
      >
        <Connect(EnvPick) />
        <Connect(HistorySearch) />
      </div>
    </div>
    <div
      className="col s12 m6 history-sections "
    >
      <div
        className="history-main history-container"
      >
        <Connect(HistoryMain) />
      </div>
    </div>
    <div
      className=" col s12 m3 history-sections "
    >
      <Connect(HistoryAna) />
    </div>
  </div>
</Fragment>
`;
